docs/ARCHITECTURE.md

```diff
@@ -224,6 +225,35 @@ The IncrementalRepairTask is the class that will perform the incremental repair
 
 [i96]: https://github.com/Ericsson/ecchronos/issues/96
 
+## Ecchronos as an Agent
+The default implementation of EcChronos was designed to be a Side-car for Cassandra, the new approach is to enable EcChronos instances to be responsible for more than one Cassandra node, at the rack, datacenter, or even entire cluster level.
+
+### Connection
+Once the instance establishes its initial connection with the node in the specified datacenter, this instance should register its control over the contact points, whether JMX or CQL, to make it clear that a single instance will be responsible for managing multiple nodes.
+Then it would be possible to keep track of what was the last time that the EcChronos instances was able to connect with a node, also for others EcChronos instances keep track about each other's health.
+
+Configuration is available on ecc.yml in the below format.
+
+```yaml
+connection:
+  cql:
+    datacenterAware:
+      enabled: true
```

Comment:
When enabling this, how does this affect the "standard" host/port setting? I think this needs to be explained, or perhaps the config should be reworked a bit?